src/sentry/runner/commands/devservices.py

```diff
@@ -48,11 +50,64 @@ def devservices():
     """
 
 
+@devservices.command()
+@click.option("--project", default="sentry")
+@click.argument("service", nargs=1)
+def attach(project, service):
+    """
+    Run a single devservice in foreground, as opposed to `up` which runs all of
+    them in the background.
+
+    Accepts a single argument, the name of the service to spawn. The service
+    will run with output printed to your terminal, and the ability to kill it
+    with ^C. This is used in devserver.
+
+    Note: This does not update images, you will have to use `devservices up`
+    for that.
+    """
+
+    os.environ["SENTRY_SKIP_BACKEND_VALIDATION"] = "1"
+
+    from sentry.runner import configure
+
+    configure()
+
+    client = get_docker_client()
+    containers = _prepare_containers(project)
+    container = _start_service(client, service, containers, project, fast=True)
+
+    def exit_handler(*_):
+        click.echo("Shutting down {}".format(service))
+        try:
+            container.stop()
+        except KeyboardInterrupt:
+            pass
+
+    atexit.register(exit_handler)
+    signal.signal(signal.SIGINT, exit_handler)
+    signal.signal(signal.SIGTERM, exit_handler)
+
+    for line in container.logs(stream=True):
+        click.echo(line, nl=False)
+
+
 @devservices.command()
 @click.option("--project", default="sentry")
 @click.option("--exclude", multiple=True, help="Services to ignore and not run.")
-def up(project, exclude):
-    "Run/update dependent services."
+@click.argument("service", nargs=1, required=False)
+def up(project, service, exclude):
```

Comment:
`service` isn't actually used in the function body.

```
joshua.li@tim-apple sentry feat/devservices-with-devserver $ sentry devservices up redis
19:30:50 [WARNING] sentry.utils.geo: settings.GEOIP_PATH_MMDB not configured.
19:30:53 [INFO] sentry.plugins.github: apps-not-configured
! Skipping bigtable due to only_if condition
! Skipping memcached due to only_if condition
> Creating 'sentry_postgres' container (listening: ('127.0.0.1', 5432))
> Pulling image 'us.gcr.io/sentryio/relay:latest'
...
```

Reply:
you're right this was dead code from an earlier experiment

Reply:
Surprised flake8 doesn't catch this.